tools/run-clang-format-in-ci.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -ex
+
+BASE_BRANCH=master
+if [[ $SYSTEM_PULLREQUEST_TARGETBRANCH ]]; then
+  git remote add upstream https://github.com/pytorch/pytorch
+  git fetch upstream "$SYSTEM_PULLREQUEST_TARGETBRANCH"
+  BASE_BRANCH="upstream/$SYSTEM_PULLREQUEST_TARGETBRANCH"
+fi
+
+# Run clang-format.
+# Exits with non-zero status if clang-format generated changes.
+time python tools/clang_format.py \
+  --verbose                       \
+  --diff "$BASE_BRANCH"           \
```

Comment:
Let's please run clang-format on all files in CI, not the diff only. Having it be diff-based in clang-tidy means that most of the time there is broken code in master and we don't know about it.